src/main/java/dk/kb/datahandler/util/HttpPostUtil.java

```diff
@@ -0,0 +1,60 @@
+package dk.kb.datahandler.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+
+/**
+ *  Util class for doing POST request with an InputStream so it is not loaded into memory
+ *  Can it be done easier with apacheIO ?  
+ */
+
+public class HttpPostUtil {
+        
+    public static String callPost(HttpURLConnection conn, InputStream input) throws IOException {
+        conn.setRequestMethod("POST");
+        conn.setConnectTimeout(30 * 1000);
+        conn.setReadTimeout(30 * 1000);
+        conn.setUseCaches(false);
```

Comment:
It seems very messy to adjust the `HttpURLConnection` both in `DsDatahandlerFacade` and in `HttpPostUtil`. Maybe let the `callPost` method take URL, MIME-type and InputStream as parameters and let it do all connection creation?